commands/blackjack.js

```diff
@@ -0,0 +1,199 @@
+const { SlashCommandBuilder, ButtonBuilder, ActionRowBuilder, ButtonStyle } = require('@discordjs/builders');
+const _ = require('lodash');
+const Deck = require('card-deck');
+const { DECK } = require('../constants');
+
+let deckOf52Cards;
+
+const convertAceToOne = (cards, cardSum) => {
+  for (var i = 0; i < cards.length; i++) {
+    if (cards[i].includes('A')) {
+      cards[i] = cards[i].replace('A', '1');
+      cardSum = cardSum - 10;
+      break;
+    }
+  }
+
+  return [cards, cardSum];
+};
+
+const getCardValue = (card) => {
+  let cardValue;
+  switch (card.substring(card.length - 1)) {
+    case 'T':
+    case 'J':
```

Comment:
oops I deleted my comment, but basically why store the value as K Q J in our deck? we can just either make the cards a numeric value (11,12,13) or instead of using stream values, we can create card objects
each card could look like this: 
 ```
{
  cardDisplayVal: "K",
  suit: "♥ ",
  cardValue: 10
}
```

Reply:
Hmm we could probably do this! It'll probably be easier to read and parse compared to an array of strings. 

But I think the hard part was maintaining the state of the player's hand. I couldn't think of a better way to maintain that other than using the `customId` which only takes a string. Open to suggestions here.

Reply:
I think we can maintain the state with a Game class? inside we can use a map or a set to keep track of everyone's hands. key used would be player id and while object stored could be array of these objects that represents the cards